packages/react-accordion/src/components/Accordion/Accordion.types.ts

```diff
@@ -8,16 +8,31 @@ export type AccordionToggleEvent<E = HTMLElement> = React.MouseEvent<E> | React.
 
 export type AccordionToggleEventHandler = (event: AccordionToggleEvent, data: AccordionToggleData) => void;
 
-export type AccordionContextValue = {
+export type AccordionCommons = {
+  /**
+   * Indicates if keyboard navigation is available
+   */
   navigable: boolean;
+  /**
+   * Indicates if Accordion support multiple Panels opened at the same time
+   */
+  multiple: boolean;
+  /**
+   * Indicates if Accordion support multiple Panels closed at the same time
+   */
+  collapsible: boolean;
+};
+
+export type AccordionContextValue = AccordionCommons & {
   /**
    * The list of opened panels by index
    */
   openItems: AccordionItemValue[];
   /**
    * Callback used by AccordionItem to request a change on it's own opened state
+   * Should be used to toggle AccordionItem
    */
-  requestToggle: AccordionToggleEventHandler;
+  requestToggle: (event: AccordionToggleEvent, data: AccordionToggleData) => void;
```

Comment:
if you're going to stop using the `AccordionToggleEventHandler` type you could probably remove it altogether

Reply:
`AccordionToggleEventHandler` is still used by `onToggle` I just decided to stop using it for the `requestToggle` method to ensure no confusion between an internal method from context and a external one for props